### PR TITLE
[NEW BONUS GUIDE] Install & enable I2P daemon + configuration + uninstall

### PR DESCRIPTION
**Pull request by twofaktor**
#### What

New bonus guide to enable I2P network adding network private redundancy

Sections added:

- Description of the service & illustration
- Install & enable the I2P daemon
- Configuration to enable only in BTC Core for the moment
- Uninstall section

### Why

This can lead to a significant improvement in reducing potential network failures in the Raspibolt nodes.

#### How

New guide document to include the new feature

#### Scope

- [ ] significant change to core configuration
- [x] independent bonus guide
- [ ] simple bug fix

Open issue: #953 

#### Test & maintenance

I tested all and is working fine for me:

`bitcoin-cli -netinfo`

```
Bitcoin Core client v23.0.0 - server 70016/Satoshi:23.0.0/

         ipv4    ipv6   onion     i2p   total   block
in          0       0      25       2      27
out         7       0       2       1      10       2
total       7       0      27       3      37

```

#### Animated GIF

giphy (1)

### CODE REVIEW

File: guide/bonus/bitcoin/i2p.md [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+i2pacceptincoming=1
+i2psam=127.0.0.1:7656
+
+# I2P node seeds
+seednode=a5qsnv3maw77mlmmzlcglu6twje6ttctd3fhpbfwcbpmewx6fczq.b32.i2p:0
+seednode=c4gfnttsuwqomiygupdqqqyy5y5emnk5c73hrfvatri67prd7vyq.b32.i2p:0
+seednode=h3r6bkn46qxftwja53pxiykntegfyfjqtnzbm6iv6r5mungmqgmq.b32.i2p:0
+seednode=jz3s4eurm5vzjresf4mwo7oni4bk36daolwxh4iqtewakylgkxmq.b32.i2p:0
+seednode=sedndhv5vpcgdmykyi5st4yqhdxl3hpdtglta4do435wupahhx6q.b32.i2p:0
+seednode=wwbw7nqr3ahkqv62cuqfwgtneekvvpnuc4i4f6yo7tpoqjswvcwa.b32.i2p:0
```
Comment (kristapsk) [inline guide/bonus/bitcoin/i2p.md:131]: This is not necessary. These are already listed in https://github.com/bitcoin/bitcoin/blob/master/contrib/seeds/nodes_main.txt, from which https://github.com/bitcoin/bitcoin/blob/master/src/chainparamsseeds.h is generated. So `bitcoind` already should know about them.

Comment (twofaktor) [reply guide/bonus/bitcoin/i2p.md]: Thanks! Fixed!

File: guide/bonus/bitcoin/i2p.md [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+i2psam=127.0.0.1:7656
+```
+
+* Restart Bitcoin Core
+
+```sh
+$ sudo systemctl restart bitcoind
+```
+
+* Wait a few minutes until Bitcoin Core started again, enter the next command to obtain your I2P address
```
Comment (kristapsk) [inline guide/bonus/bitcoin/i2p.md:130]: It's not only until Bitcoin Core has started again. There is usually some additional delay with I2P local address appearing after Bitcoin Core already has connected to onion peers. Maybe should mention that.

File: guide/bonus/bitcoin/i2p.md [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+
+---
+
+## I2P
+
+[I2P](https://geti2p.net/en/){:target="_blank"} is a universal anonymous network layer. All communications over I2P are anonymous and end-to-end encrypted, participants don't reveal their real IP addresses. I2P allows people from all around the world to communicate and share information without restrictions.
+
+I2P client is a software used for building and using anonymous I2P networks. Such networks are commonly used for anonymous peer-to-peer applications (filesharing, cryptocurrencies) and anonymous client-server applications (websites, instant messengers, chat-servers).
+
+We are to use [i2pd](https://i2pd.readthedocs.io/en/latest/) (I2P Daemon) a full-featured C++ implementation of the I2P client as a Tor network complement.
```
Comment (kristapsk) [inline guide/bonus/bitcoin/i2p.md:23]: nit
```suggestion
We are to use [i2pd](https://i2pd.readthedocs.io/en/latest/) (I2P Daemon), a full-featured C++ implementation of the I2P client as a Tor network complement.
```

**Comment by kristapsk** [comment]
Apart from nits above, tACK from me, except for uninstall part.

File: guide/bonus/bitcoin/i2p.md (hunk 2 above)
Comment (twofaktor) [reply guide/bonus/bitcoin/i2p.md]: Fixed

File: guide/bonus/bitcoin/i2p.md (hunk 3 above)
Comment (twofaktor) [reply guide/bonus/bitcoin/i2p.md]: Fixed

**Comment by twofaktor** [comment]
Fixed, re-review

File: guide/bonus/bitcoin/i2p.md [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+* Delete or comment the next lines in `bitcoin.conf` file. Save and exit
+
+```sh
+$ sudo nano /data/bitcoin/bitcoin.conf
+```
+
+```sh
+$ sudo nano /data/bitcoin/bitcoin.conf
+```
+
```
Comment (VajraOfIndra) [inline guide/bonus/bitcoin/i2p.md:180]:
```suggestion
```

Duplicated command

**Review by VajraOfIndra — APPROVED** [review]
LGTM, tACK (except the uninstall section, utACK)

Thanks for this awesome addition :+1:

**Comment by twofaktor** [comment]
Thanks for yours reviews guys! 

When you say tACK or tACK, do it mean that I have to take action somehow?

What mean tACK and untACK? 

Sorry for my question, I'm noob in Github yet 🙏

Thanks again

**Comment by twofaktor** [comment]
Thanks for yours reviews guys! 

When you say tACK or tACK, do it mean that I have to take action somehow?

What mean tACK and untACK? 

Sorry for my question, I'm noob in Github yet 🙏

Thanks again


Thanks for the explanations,

Let me arrive to my desk and I change this

**Comment by twofaktor** [comment]
Fixed duplicate command & standardized format. Ready for the merge! Let's go! 🚀

File: guide/bonus/bitcoin/i2p.md (hunk 4 above)
Comment (twofaktor) [reply guide/bonus/bitcoin/i2p.md]: Fixed!!

**Pull request merged**